src/GoogleFitOnFhir/Clients/GoogleFit/GoogleFitClientContext.cs

```diff
@@ -29,9 +29,8 @@ public GoogleFitClientContext(string clientId, string clientSecret, string hostN
                 "https://www.googleapis.com/auth/userinfo.email",
                 "https://www.googleapis.com/auth/userinfo.profile",
                 FitnessService.Scope.FitnessBloodGlucoseRead,
-                FitnessService.Scope.FitnessBloodGlucoseWrite,
                 FitnessService.Scope.FitnessHeartRateRead,
-                FitnessService.Scope.FitnessHeartRateWrite,
+                FitnessService.Scope.FitnessBloodPressureRead,
```

Comment:
I think for now, we should just add all of the read scopes to ensure we are collecting all of the test data possible. Here is the list I am using:

"https://www.googleapis.com/auth/userinfo.email",
"https://www.googleapis.com/auth/userinfo.profile",
FitnessService.Scope.FitnessActivityRead,
FitnessService.Scope.FitnessSleepRead,
FitnessService.Scope.FitnessReproductiveHealthRead,
FitnessService.Scope.FitnessOxygenSaturationRead,
FitnessService.Scope.FitnessNutritionRead,
FitnessService.Scope.FitnessLocationRead,
FitnessService.Scope.FitnessBodyTemperatureRead,
FitnessService.Scope.FitnessBodyRead,
FitnessService.Scope.FitnessBloodPressureRead,
FitnessService.Scope.FitnessBloodGlucoseRead,
FitnessService.Scope.FitnessHeartRateRead,